src/components/organisms_new/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -22,34 +25,42 @@ const NewPaneManagerLeftMenu: React.FC = () => {
   );
 
   return (
-    <ActivityBar
-      activities={activities}
-      extraActivities={extraActivities}
-      isActive={isActive}
-      value={leftMenuSelection}
-      extraValue={leftMenuBottomSelection}
-      onChange={activityName => {
-        if (activityName === leftMenuSelection && leftActive) {
-          dispatch(setLeftMenuIsActive(false));
-          return;
-        }
-
-        if (!leftActive) {
-          dispatch(setLeftMenuIsActive(true));
-        }
-
-        dispatch(setLeftMenuSelection(activityName));
-      }}
-      onChangeExtra={activityName => {
-        if (leftMenuBottomSelection === activityName) {
-          dispatch(setLeftBottomMenuSelection(undefined));
-          return;
-        }
-
-        dispatch(setLeftBottomMenuSelection(activityName));
-      }}
-    />
+    <Container>
+      <ActivityBar
+        activities={activities}
+        extraActivities={extraActivities}
+        isActive={isActive}
+        value={leftMenuSelection}
+        extraValue={leftMenuBottomSelection}
+        onChange={activityName => {
+          if (activityName === leftMenuSelection && leftActive) {
+            dispatch(setLeftMenuIsActive(false));
+            return;
+          }
+
+          if (!leftActive) {
+            dispatch(setLeftMenuIsActive(true));
+          }
+
+          dispatch(setLeftMenuSelection(activityName));
+        }}
+        onChangeExtra={activityName => {
+          if (leftMenuBottomSelection === activityName) {
+            dispatch(setLeftBottomMenuSelection(undefined));
+            return;
+          }
+
+          dispatch(setLeftBottomMenuSelection(activityName));
+        }}
+      />
+    </Container>
   );
 };
 
 export default NewPaneManagerLeftMenu;
+
+const Container = styled.div`
```

Comment:
Can't we just use the `style` prop for the `ActivityBar` directly in here?

Reply:
We could use it if we added a class name or made it accept the style prop